pulsar/consumer_test.go

```diff
@@ -4030,9 +4030,6 @@ func TestConsumerWithAutoScaledQueueReceive(t *testing.T) {
 	_, err = c.Receive(context.Background())
 	assert.Nil(t, err)
 
-	// waiting for prefetched message passing from queueCh to dispatcher()
-	time.Sleep(time.Second)
```

Comment:
CI failed, it seems we can't remove this.

Reply:
You can use `retryAssert` instead of sleep.

https://github.com/apache/pulsar-client-go/blob/e2ea255052e8a527091791ef368851d885ee2d45/pulsar/helper_for_test.go#L176-L185